data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -188,6 +188,10 @@ sudo podman run -d --net host --privileged --name image-customization \
     -v /etc/containers:/tmp/containers:z \
     ${CUSTOMIZATION_IMAGE}
 
+{{ if .PlatformData.BareMetal.ProvisioningIPv6 }}
+IP_OPTIONS="ip=dhcp6"
```

Comment:
Does this do the right thing if the Provisioning network is IPv4 and the external network is IPv6? Or can this not happen?

If they're separate I think we should rename the two variables to PROVISIONING_IP_OPTIONS and EXTERNAL_IP_OPTIONS to avoid confusion.